apps/js-machine-front/src/app/stores/ui.store.ts

```diff
@@ -1 +1,13 @@
-export class UiStore {}
+import { action, observable, runInAction } from "mobx";
+import { RootStore } from "@js-machine-app/front/stores/root.store";
+
+export class UiStore {
+  @observable public isLoading?: boolean;
+
+  public constructor(private rootStore: RootStore) {
+  }
+
+  @action public setIsLoading =  (isLoading: boolean) => {
+    runInAction(() => (this.isLoading = isLoading));
```

Comment:
you don't need runInAction if your method is synchronous and it marked as @action

Reply:
fixed, thanks